drush/Commands/cgov/CgovPdqGlossifyCommands.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Drush\Commands\cgov;
+
+use Drush\Commands\DrushCommands;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * A Drush commandfile to load pdq glossifier.
+ */
+class CgovPdqGlossifyCommands extends DrushCommands {
+
+  use StringTranslationTrait;
+
+  /**
+   * Glossifier data based on an input file.
+   *
+   * @param string $glossifierFile
+   *   Path to the json file containing the glosiify information.
+   *
+   * @command cgov:load-glossifier
+   * @aliases load-glossifier, cgpdg
+   * @usage drush cgov:load-glossifier /tmp/glossifier_refresh.json
+   *   Process json content for glossifier
+   * @bootstrap full
+   */
+  public function loadPdqGlossifier($glossifierFile) {
+    $count = NULL;
+    if (!file_exists($glossifierFile)) {
+      throw new \Exception(dt('Unable to load file !file.', ['!file' => $glossifierFile]));
+    }
+    $jsonData = file_get_contents($glossifierFile);
+    if (!empty($jsonData)) {
```

Comment:
If the data is empty this should also be an `error` message and we should exit. Since this is new code, we should have hard errors. The existing API code we want to keep exactly the same.